src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/PluginsConfigurationHelper.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.Configuration
+{
+    internal static class PluginsConfigurationHelper
+    {
+        public static TracerProviderBuilder InvokePlugins(this TracerProviderBuilder builder, IEnumerable<string> pluginsAssemblyQualifiedNames)
+        {
+            foreach (var assemblyQualifiedName in pluginsAssemblyQualifiedNames)
+            {
+                builder = builder.InvokePlugin(assemblyQualifiedName);
+            }
+
+            return builder;
+        }
+
+        private static TracerProviderBuilder InvokePlugin(this TracerProviderBuilder builder, string pluginAssemblyQualifiedName)
+        {
+            // get the type and method
+            var t = Type.GetType(pluginAssemblyQualifiedName);
```

Comment:
I assume if we get exception here, crashing app is exactly what we want to happen (fail fast)?

Reply:
I think so. Maybe it would be good to catch it and make some more readable error, but I did not want to waste time on it as this is a PoC.